Examples/tvOS Players CollectionView/tvOS Players CollectionView/PlayersCollectionViewController.swift

```diff
@@ -0,0 +1,130 @@
+                    updatePlaybackLabel()
+                }
+            }
+
+            
+            let systemPVC = SystemPlayerViewController()
+            systemPVC.player = player
+            models[index].systemPlayerViewController = systemPVC
+            
+            collectionView?.reloadItems(at: [IndexPath(row: index, section: 0)])
+        case (.error(let error), _), (_, .error(let error)):
+            let alert = UIAlertController(title: "Error",
+                                          message: "\(error)",
+                preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: "OK",
+                                          style: .default,
+                                          handler: nil))
+            present(alert, animated: true, completion: nil)
+        default: break
+        }
+    }
+    
+    override func collectionView(_ collectionView: UICollectionView,
+                                 numberOfItemsInSection section: Int) -> Int {
+        return models.count
+    }
+    
+    override func collectionView(_ collectionView: UICollectionView,
+                                 cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "PlayersCollectionCellId",
+            
+                                                      for: indexPath)
+        guard
+            let controller = models[indexPath.row].systemPlayerViewController,
+            let playerCell = cell as? PlayersCollectionViewCell else { return cell }
+        
+        addChildViewController(controller)
+        controller.view.frame = CGRect(x: 0,
+                                       y: 0,
+                                       width: cell.contentView.frame.width,
+                                       height: cell.contentView.frame.height)
+        cell.contentView.addSubview(controller.view)
+        controller.didMove(toParentViewController: self)
+        
+        playerCell.removePlayerViewController = {
+            controller.willMove(toParentViewController: nil)
+            controller.view.removeFromSuperview()
+            controller.removeFromParentViewController()
+        }
+        
+        return playerCell
+    }
+    
+    override func collectionView(_ collectionView: UICollectionView,
+                                 canFocusItemAt indexPath: IndexPath) -> Bool {
+        return false
+    }
+    
+    override func collectionView(_ collectionView: UICollectionView,
+                                 didEndDisplaying cell: UICollectionViewCell,
+                                 forItemAt indexPath: IndexPath) {
+        (cell as? PlayersCollectionViewCell)?.removePlayerViewController?()
+    }
+}
+
+extension PlayersCollectionViewController {
+    struct Model {
+        let videoId: String
+        var systemPlayerViewController: SystemPlayerViewController?
+        var playbackIsReady: Bool
```

Comment:
What is the reason of storing `isPlaybackReady` in the model?
The value could be directly used without storing it in `Model`.